SpatialGDK/Source/SpatialGDKServices/Public/SpatialCommandUtils.h

```diff
@@ -3,6 +3,7 @@
 #pragma once
 
 #include "CoreMinimal.h"
+#include "HAL/PlatformProcess.h"
```

Comment:
do we need that extra include for the loctext?

Reply:
No this one is for FProcHandle that's returned by one of the functions.